var/spack/repos/builtin/packages/krims/package.py

```diff
@@ -0,0 +1,70 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Krims(CMakePackage):
+    """The bucket of Krimskrams every C or C++ project needs"""
+
+    maintainers = ['mfherbst']
+
+    homepage = "http://lazyten.org/krims"
+    url      = "https://github.com/lazyten/krims/archive/v0.2.1.tar.gz"
+
+    version("0.2.1", "bf219656fd81c342a4782bad7d7beed0")
+    version("develop", git="https://github.com/lazyten/krims.git",
+            branch="master")
+
+    variant("build_type", default="DebugRelease",
+            description="The build type to build",
+            values=("Debug", "Release", "DebugRelease"))
```

Comment:
This comes automatically from `CMakePackage`, so why did you add that? Does this package doesn't support `MinSizeRel`?

Reply:
`krims` comes with the CMake module [DebugReleaseBuild](https://github.com/lazyten/krims#cmake-module-debugreleasebuild), which drops support for `RelWithDebInfo` as well as `MinSizeRel` in favour of a simultanious build of Debug and Release (`DebugRelease`). That system is also employed for `lazyten`, hence we override the `build_type` variant there as well.